CHANGELOG.md

```diff
@@ -3027,3 +3027,8 @@ This is a full list of proofs that have changed form to use irrelevant instance
   ```agda
   <-weakInduction-startingFrom : P i →  (∀ j → P (inject₁ j) → P (suc j)) → ∀ {j} → j ≥ i → P j
   ```
+
+* In `Data.List.Relation.Binary.Permutation.Propositional.Properties`:
+  ```agda
+  foldr-cmonoid : ∀ {xs ys} → xs ↭ ys → foldr _∙_ ε xs ≈ foldr _∙_ ε ys
```

Comment:
We can omit the implicit parameters in the entry. However we do need the `CommutativeMonoid` argument here.

Reply:
I removed ∀ {xs ys}